contracts/LockedNORI.sol

```diff
@@ -349,14 +374,38 @@ contract LockedNORI is
   }
 
   /**
-   * @dev Vested balance less any claimed amount at current block timestamp.
+   * @notice Vested balance less any claimed amount at current block timestamp.
    */
   function vestedBalanceOf(address account) external view returns (uint256) {
     return _vestedBalanceOf(account, block.timestamp);
   }
 
+  // todo document expected initialzation state
+  function initialize(BridgedPolygonNORI bridgedPolygonNoriAddress)
```

Comment:
Can we move this back down to the original place to reduce the noise in the diff?